### PR TITLE
Initial implementation of downloads page

### PR DESCRIPTION
**Pull request by Mrtenz**

### CODE REVIEW

**Comment by Mrtenz** [comment]
[sc-8890]

File: src/constants.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import secondaryFeaturesSlide0 from './assets/images/secondary-features-slide-0.svg';
+import secondaryFeaturesSlide1 from './assets/images/secondary-features-slide-1.svg';
+import secondaryFeaturesSlide2 from './assets/images/secondary-features-slide-2.svg';
+import gettingStartedSlide0 from './assets/images/getting-started-slide-0.svg';
+import gettingStartedSlide1 from './assets/images/getting-started-slide-1.svg';
+import gettingStartedSlide2 from './assets/images/getting-started-slide-2.svg';
+
+/**
+ * URL for the old desktop application.
+ * TODO: Figure out where this should link to.
```
Comment (FrederikBolding) [inline src/constants.ts]: ?

File: src/hooks/useVersion.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    fetch(`${GITHUB_API_ENDPOINT}/repos/${site.siteMetadata.repository}/releases?per_page=100`, {
+      headers: {
+        Accept: 'application/vnd.github.v3+json'
+      }
+    })
+      .then((response) => response.json())
+      .then((json: Release[]) =>
+        json
+          .filter((release) => !!valid(coerce(release.tag_name)))
+          .filter((release) => release.prerelease)
```
Comment (FrederikBolding) [inline src/hooks/useVersion.ts:43]: Add comment that this might need to be removed.

File: src/theme.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { theme as defaultTheme } from '@mycrypto/ui';
+import merge from 'lodash.merge';
+
+export const theme = merge(defaultTheme, {
```
Comment (FrederikBolding) [inline src/theme.tsx]: This file can just be a `.ts`

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ steps.nvm.outputs.NVMRC }}
+
+      - name: Cache Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+          key: ${{ hashFiles('yarn.lock') }}-${{ hashFiles('config/**/*') }}
```
Comment (FrederikBolding) [inline .github/workflows/ci.yml]: Can we just do `${{ hashFiles('yarn.lock') }}`

File: src/components/DownloadLinks.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+import type { FunctionComponent } from 'react';
+
+import apple from '../assets/images/apple.svg';
+import linux from '../assets/images/linux.svg';
+import windows from '../assets/images/windows.svg';
+
+// TODO: Actually link the buttons to the download
+export const DownloadLinks: FunctionComponent = () => (
+  <Box>
+    <Button variant="inverted" fontWeight="bold" height="56px" marginRight="3">
```
Comment (FrederikBolding) [inline src/components/DownloadLinks.tsx]: Alignment looks a bit off here

File: src/components/Overview.tsx [hunk 6]
```diff
@@ -0,0 +1,251 @@
+import { t, Trans } from '@lingui/macro';
+import { Body, Box, Icon, Image } from '@mycrypto/ui';
+import type { FunctionComponent } from 'react';
+
+import exodus from '../assets/images/exodus.svg';
+import metamask from '../assets/images/metamask.svg';
+import mycrypto from '../assets/images/mycrypto.svg';
+import quill from '../assets/images/quill.svg';
+
+export const Overview: FunctionComponent = () => (
```
Comment (FrederikBolding) [inline src/components/Overview.tsx:10]: Please review these with Jordan to make sure these are correct

File: src/components/Footer.tsx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+              <Link to={link.url} isExternal={true} color="inherit">
+                {link.title}
+              </Link>
+            </FooterLink>
+          ))}
+        </FooterLinkColumn>
+      ))}
+    </FooterLinks>
+    <FooterSeparator order={3} />
+    <FooterDonateAndSubscribe listId="foo" tag="download.mycrypto.com" />
```
Comment (FrederikBolding) [inline src/components/Footer.tsx]: Do we still want the donation addresses here?

File: src/constants.ts [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+  },
+  {
+    title: t`Encrypt your keys`,
+    image: secondaryFeaturesSlide1,
+    text: t`Quill encrypts your keys for you so you can store your wallets confidently.`
+  },
+  {
+    title: t`Thrive offline`,
+    image: secondaryFeaturesSlide2,
+    text: t`Quill is fully functional offline. Airgap your device for even more cold storage security!`
```
Comment (FrederikBolding) [inline src/constants.ts:52]: This is a bit long and causes the page to jump

File: src/constants.ts [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+  },
+  {
+    title: t`Setup your accounts`,
+    image: gettingStartedSlide1,
+    text: t`Generate a new 12-word secret recovery phrase and/or import and existing wallet.`
+  },
+  {
+    title: t`Connect to MyCrypto`,
+    image: gettingStartedSlide2,
+    text: t`Visit MyCrypto.com and connect to your new Quill wallet. Now you can send transaction requests from MyCrypto directly to Quill!`
```
Comment (FrederikBolding) [inline src/constants.ts:70]: This is a bit long and causes the page to jump

File: src/components/Slideshow.tsx [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+  slides: Slide[];
+}
+
+export const SlideshowButton: FunctionComponent<SlideshowButtonProps> = ({
+  title,
+  isActive,
+  onClick,
+  index
+}) => {
+  const handleClick = () => {
```
Comment (FrederikBolding) [inline src/components/Slideshow.tsx]: Can we remove `{}`?

File: src/components/Slideshow.tsx [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+        return setSlide((state) => state + 1);
+      }
+
+      setSlide(0);
+    }, 7500);
+
+    return () => clearTimeout(timeout);
+  }, [slide]);
+
+  const handleClick = (index: number) => setSlide(index);
```
Comment (FrederikBolding) [inline src/components/Slideshow.tsx:63]: Looks like this clears the timeout, can you confirm?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/components/Slideshow.tsx (hunk 11 above)
Comment (Mrtenz) [reply src/components/Slideshow.tsx:63]: Yes it does, as intended.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**